ember/app/styles/partials/_vars.scss

```diff
@@ -19,6 +19,11 @@ $color_font-dark: #091F2F;
 $color_state-negative: #db1e1e;
 $color_state-negative--dark: darken($color_state-negative, 15);
 
+$color_button-disabled: #b3b3b3;
+
 $font_family-primary: 'Nunito', Helvetica, sans-serif;
 
 $transition_border: border-color .25s;
+
+$development_font_size-h1: 26px;
+$development_font_size-h2: 18px;
```

Comment:
To reduce indirection, my typical convention for this file is to globally scope generic constants that don't have any affiliation to a single piece of the app. I'd place these two variables at the top of the file which they are being used, scoped inside of that file's wrapping class. 

```scss
.component.development-form {
    $font_size-h1: 26px;
    $font_size-h2: 18px;

    ...
```